src/Model/API/EdgeScripting/EdgeScript/ListEdgeScripts.php

```diff
@@ -33,6 +33,7 @@ public function getHeaders(): array
     public function getQuery(): array
     {
         return [
+            new AbstractParameter(name: 'type', type: Type::ARRAY_TYPE),
```

Comment:
Update documentation

Reply:
Maybe this should have been the following:

```suggestion
            new AbstractParameter(name: 'type', type: Type::ARRAY_TYPE, children: [
                new AbstractParameter(name: null, type: Type::INT_TYPE),
            ]),
```

Thinks without children it's still okay (needs testing) but would be more explicit if it was the above. This might need to be updated in the generator.